include/seqan3/alphabet/gap/gap.hpp

```diff
@@ -139,9 +143,16 @@ struct gap
     {
         return value >= rhs.value;
     }
-    //!@}
+    //!\}
+
+    //!\privatesection
+    //!\brief The data member.
+    static constexpr rank_type value = 0;
```

Comment:
why do we even have this? We could just hardcode `to_rank` to return 0, like `to_char` is hardcoded to return `-`. (also hardcode the comparison operators)...